code/drasil-code/lib/Language/Drasil/Choices.hs

```diff
@@ -74,60 +39,101 @@ class RenderChoices a where
     showChsList :: [a] -> Sentence
     showChsList lst = foldlSent_ (map showChs lst)
 
+-- | Architecture of a program
+data Architecture = Archt {
+  modularity :: Modularity,
+  impType :: ImplementationType
+}
+-- | Constructor to create a Architecture
+makeArchit :: Modularity -> ImplementationType -> Architecture
+makeArchit = Archt
+
 -- | Modularity of a program.
-data Modularity = Modular InputModule -- ^ Different modules. For controller, 
-                                      -- input, calculations, output.
-                | Unmodular -- ^ All generated code is in one module/file.
+data Modularity = Modular InputModule
+                | Unmodular
 
 -- | Renders the modularity of a program.
-instance RenderChoices Modularity where 
+instance RenderChoices Modularity where
   showChs Unmodular = S "Unmodular"
   showChs (Modular Combined) = S "Modular Combined"
   showChs (Modular Separated)= S "Modular Separated"
 
 -- | Options for input modules.
-data InputModule = Combined -- ^ Input-related functions combined in one module.
```

Comment:
Here (and above), you're removing comments from the code. Why? This is usually not a good idea?

Reply:
I think I misunderstood the reason add the wiki link to the comments. I thought I have to remove comments which are already documented in the wiki. 

All removed comments occurred in the Choices file, I can add them back.

Reply:
If we had Drasil fully working, we wouldn't want such redundancy. But in the meantime, we don't want to have people have to hunt for documentation.

Reply:
I added previous comments back